lib/utils/isStandardSyntaxKeyframesName.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const hasInterpolation = require('./hasInterpolation');
+
+/**
+ * Check whether a keyframes name is standard
+ *
+ * @param {string} keyframesName
+ * @returns {boolean}
+ */
+
+module.exports = function (keyframesName) {
+	// SCSS or Less interpolation
```

Comment:
```suggestion
```

We can loose the comment. (The `hasInterpolation` checks for more than just Less or Sass interpolation).